.travis.yml

```diff
@@ -235,32 +235,39 @@ matrix:
         - USE_SYSLIBS=false
         - SHARED_LIBSCSYNTH=false
 
-    # macOS 10.14 xcode10
+    # macOS 10.13 xcode10 legacy build with qt 5.9
     - os: osx
-      osx_image: xcode10.3
+      osx_image: xcode10.1
       env:
-        - JOB_NAME="macos xcode10.3"
+        - JOB_NAME="macos xcode10.1 legacy build with qt 5.9"
+        - UPDATE_HOMEBREW=false
+        - QT_FORMULA="supercollider/formulae/qt@5.9.3"
```

Comment:
change UPDATE_HOMEBREW to IS_LEGACY_BUILD, remove QT_FORMULA, and just use the name of the formula directly in the script

Reply:
This was intentional. My idea was that we would have control over updating homebrew in the configuration script, so we   could easily turn in on/off for individual builds if desired. Similarly, specifying QT formula in the script would make it possible to choose a separate formula for other builds - probably only in the situations like with Qt 5.15.1, but still I thought it would be nice to have this configurability.
If you still think it should be changed to a single variable and the formula being hardcoded in the script, I'm happy to change this.

Reply:
yes, please change it. this does not seem like a likely enough future that we need to make the existing code more complicated